HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,302 @@
+created repositories.
+
+## GitHub Actions 
+Next to the issues, each of the repositories uses GitHub Actions for continuous integration.
+While the GitHub Action workflow definitions are stored as part of the repository, we use custom action runners on
+hardware in the Google Cloud.
+The default GitHub action runners can not be used due to the complexity of `PROJECT`, especially for the services sub-project.
+To continue the development of `PROJECT` based on best practices and continuous integration, we propose to set up
+clones of the current hardware and action runners.
+The Hedera Hashgraph projects define teams with specific authorities.
+Those configurations should be migrated, and maintainers and committers for each sub-project should be defined.
+The complete list of GitHub users that should be added as maintainers or committers to the
+sub-projects will be submitted later.
+
+## Documentation
+The Hedera project provides a repository for the documentation of the Hedera network, including its deployment and
+utilization of ever-evolving services.
+That repository can be found at https://github.com/hashgraph/hedera-docs.
+The content of the repository uses gitbook to create complete documentation that is currently hosted
+at https://docs.hedera.com/hedera.
+We propose to create a neutral hosted version of the documentation at https://docs.PROJECT.com, removing or modifying
+existing content that is Hedera specific.
+We will work on that directly after the setup to remove all those parts from the transited repository.
+
+## DCO
+We suggest setting up a DCO process by LFX for all `PROJECT` repositories.
+The DCO documents will be based on the LF templates.
+We will provide the final DCO documents separately.
+
+## HIP
+The https://github.com/hashgraph/hedera-improvement-proposal repository contains the Hedera improvement proposals (HIP)
+of Hedera.
+For `PROJECT` we need improvement proposals and a process for them, too.
+Here, the HIP repo should be migrated to https://github.com/PROJECT/PROJECT-improvement-proposal.
+Today a frontend showing all HIPs is hosted at https://hips.hedera.com/.
+We propose to host a clone of that at https://NAME.PROJECT.com. 
+
+## Build
+As you can see in this chapter, `PROJECT` contains many sub-projects that need specific build steps and infrastructure.
+Therefore, it is important to invest in `PROJECT`’s build, release, and deployment.
+The goal must be to have a vendor-neutral build and release process led by LF.
+We propose to nominate a release manager by LF who is involved in the project’s transition as soon as possible.
+
+## Slack
+With all the given sub-projects and the big community around `PROJECT`, it will be a must to have an open and easy way
+to communicate with the community.
+Therefore, we propose to create a `PROJECT` Slack.
+
+## GitHub applications
+We have several GitHub applications that are used in the repositories today.
```

Comment:
Do you have a list of these applications which are currently in use?

Reply:
I will create a list and add it to the proposal

Reply:
All applications that are current used in the hashgraph org:

CircleCI Checks
Codacy Production
Codecov
Common Room
DCO
[Gitbook.com](http://gitbook.com/)
Linear
[monday.com](http://monday.com/)
Netlify
Notion
Semgrep-app
Slack
SonarCloud
StepSecurity Actions Security
Terraform Cloud
Vercel
WIP
Wiz Inc.

Reply:
@ryjones : Can you comment on use of the above applications within LF and LFDT?